common/OpTestHMC.py

```diff
@@ -55,6 +55,42 @@
 BOOTTIME = 500
 STALLTIME = 5
 
+class OpSecureBootUtilities():
+    '''
+    This class addresses the secureboot related pre-requisites.
+    '''
+
+    def __init__(self, console=None, distro=None):
+        self.console = console
+        self.distro = distro
+
+    def check_kernel_signature(self):
+        vmlinux = "vmlinuz"  # RHEL
+        if self.distro == 'SLES':
```

Comment:
I think you have to add one more string like suse AFAIK, please re-check on this

Reply:
Done